apps/frontend/vue.config.js

```diff
@@ -12,6 +12,17 @@ const changelog = parsed.changelog || '';
 const branch = parsed.branch || '';
 const issues = parsed.issues || '';
 
+// This grabs the css for Bootstrap to allow for HTML export
+fs.copyFile(
```

Comment:
Would this method work for the icons as well? It looks like `@mdi/font` is just a webfont and a CSS file.

Reply:
There doesn't seem to be a way to embed the webfont inside of the HTML. I've been trying to embed the icons as SVG in CSS but that isn't working out so far.